src/handlers/deeplinks.ts

```diff
@@ -39,8 +47,8 @@ export default async function handleDeeplink(
   const urlObj = new URL(url);
   if (urlObj.protocol === 'ethereum:') {
     ethereumUtils.parseEthereumUrl(url);
-  } else if (urlObj.protocol === 'https:') {
-    const action = urlObj.pathname.split('/')[1];
+  } else if (urlObj.protocol === 'https:' || urlObj.protocol === 'rainbow:') {
+    const action = urlObj.host;
```

Comment:
Talked offline, but just noting for others that we need a better solution here for parsing these URLs, since some have hosts, like `rnbwapp.com`, and some don't, like `rainbow://token?xxx`

Reply:
added a follow up ticket here with some additional scope: https://linear.app/rainbow/issue/APP-89/standardize-scanner-deeplink-logic